src/peer-store/book.js

```diff
@@ -75,9 +197,17 @@ class Book {
       [this.eventProperty]: []
     })
 
+    // Update Persistence datastore
+    this._persistData(peerId, [])
```

Comment:
Maybe we should remove it instead 🤔

Reply:
Yes, persist the delete action. Otherwise we'll bloat the datastore with peers that we "deleted"